src/components/screens/IndexScreen/IndexScreen.js

```diff
@@ -116,6 +121,7 @@ export function IndexScreen({
 
   return (
     <>
+      <GlobalStyle />
```

Comment:
Why is this necessary?

Reply:
It's needed here now that the index template file has been deleted
https://github.com/storybookjs/frontpage/pull/428#discussion_r1033651834